src/syrupy/__init__.py

```diff
@@ -87,15 +87,16 @@ def pytest_assertrepr_compare(
     https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_assertrepr_compare
     """
     with __terminal_color(config):
+        recieved_name = received_style("[+ received]")
+
+        def snapshot_name(name: str) -> str:
+            return snapshot_style(f"[- {name}]")
```

Comment:
Opportunistic formatting change?

Reply:
Yup, easier to see which diff line is related to what, both for accessibility and when color output is disabled